test/nonregression/iotools/test_run_converters.py

```diff
@@ -40,23 +32,33 @@ def test_name(request):
     return request.param
 
 
+@pytest.mark.skip(reason="Reference dataset not up to date")
```

Comment:
That's because we are not using `pytest` idiomatically. We are not letting `pytest` discover tests (and honour annotations such as `skip`) and instead we have implemented our own [test runner](https://github.com/aramis-lab/clinica/blob/ce39b62986a2ccd4c2c7f3f65304ed4fc3f77a43/test/nonregression/iotools/test_run_converters.py#L171).

This annotation is useless.